doc/tutorial/c_tutorial/c_tutorial.rst

```diff
@@ -0,0 +1,487 @@
+************************
+Integrating C with DDlog
+************************
+
+.. contents::
+
+Prerequisites
+=============
+
+First, you need to obtain DDlog_ from the official repository, which contains instructions how to install and set it up.
+There is also an expectation that you are familiar with DDlog syntax and commands, i.e., you have read the tutorial or at least skimmed through it.
+
+.. _DDlog: https://github.com/vmware/differential-datalog
+
+Introduction
+============
+
+In this set of tutorials we are going to explore various features of DDlog programming language and learn how to integrate DDlog program into C program.
+To do that we are going to build a simple yet functional application as it is more fun to build real programs.
+We will start by writing a simple program with a limited functionality to get familiar with the basics of DDlog.
+Afterwards, over the course of this set of tutorials, we will gradually improve our application by applying more advanced techniques offered by DDlog.
+
+Tutorial #1 - Reachability Monitor
+==================================
+
+In our first tutorial we are going to build a program that will calculate the reachability information in the network.

```

Comment:
"in a network composed of nodes connected by links."
You are using nodes and links in the next phrase without having defined them.